freight/notifiers/github.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+
+__all__ = ['GithubNotifier']
+
+from flask import current_app
+
+from freight import http
+from freight.models import App, Task, TaskStatus
+
+from .base import Notifier, NotifierEvent
+
+
+class GithubNotifier(Notifier):
+    def get_options(self):
+        return {
+            'repo': {'required': True},
+            'api_root': {'required': False},
+        }
+
+    def send_deploy(self, deploy, task, config, event):
+        token = current_app.config['GITHUB_TOKEN']
+        if not token:
+            raise ValueError('GITHUB_TOKEN is not set')
+        headers = {
+            'Accepts': 'application/json',
```

Comment:
`Accept`, and you should use `application/vnd.github.v3+json` to explicitly say which API version.